client/src/store/index.js

```diff
@@ -408,8 +408,8 @@ export default new Vuex.Store({
             state.compInfo.compNames.compName1.decision_text = record.decision_text
             state.compInfo.compNames.compName1.comment = record.comment
 
-            // if this name is not yet examined, set it as current name
-            if (record.state == 'NE') {
+            // if this name is not yet examined or it is approved (in case of reset/re-open), set it as current name
```

Comment:
Would it be clearer to say in the comment,  'it was previously approved but reset/reopen' ?

Reply:
Actually upon looking at this again, I think it's a bug - not sure this should only be approved, it might also be rejected and conditional. Let me check into this...